com/metacube/DSA/Assignment3/Bowler.java

```diff
@@ -0,0 +1,50 @@
+package GET2018.com.metacube.DSA.Assignment3;
+
+/**
+ * Copyright (c) 2018 Metacube.com. All rights reserved.
+ * Implementation of Bowler Class where we describe about properties of bowler
+ * @author Chirag Jain
+ * 
+ */
+public class Bowler {
+	int bowl;
+	String name;
+	
+	public Bowler(int bowl, String name) throws CustomException {
+		if(bowl == 0) {
+			throw new CustomException("Bowler not allowed with zero bowl");
+		}
+		
```

Comment:
Its not a good practice to throw exceptions from the constructor. Constructor is basically to assign values or set defaults. For your purpose you can create a setter method.